### ISSUE TITLE
[Test failure] Dark Mode Compatibility Issues: Visibility Problems Across Several Sections

### ISSUE THREAD

**Issue by HeliaEnayati** [openedx/wg-build-test-release#410]
### Release

sumac

### Expected behavior

When dark mode is enabled, all parts of the interface should be readable and fully compatible with dark mode settings.

### Actual behavior

- Handouts and Updates: These sections have a white background.

![image](https://github.com/user-attachments/assets/c4da4004-4605-4b9e-95a4-6d8d73d44b05)

- Notes: The course tabs to become unreadable. The notes background is white, making it impossible to read the notes' descriptions or the notes 
![Screenshot from 2024-11-10 15-35-55](https://github.com/user-attachments/assets/a18ac0d5-b0d6-45ce-8a15-fa25a819ed71)

- Wiki Settings: The group dropdown menu is illegible in dark mode because both the text and background are white.

- Textbook: Similar issues to the notes section, where course tabs are unreadable. Additionally, the area displaying chapter names has a white background
![image](https://github.com/user-attachments/assets/fe5f52b9-2a14-463d-8adc-6137afe1e516)

- Teams: When you navigate to the Teams page as a learner, the description color, "page-content-nav," all dropdowns, and team cards are not displaying correctly. Additionally, these issues occur when trying to create a new team.
![image](https://github.com/user-attachments/assets/b2505a54-37b2-446d-976c-0c79dbb7648b)

### Steps to reproduce

1. Add Handouts and Updates
2. Add notes
3. Add wiki 
4. Add textbook
5. Add teams

**Comment by magajh** [comment]
Thank you for the report! An issue has been opened for the Tutor-Indigo maintainers to look into and address this problem https://github.com/overhangio/tutor-indigo/issues/107

**Comment by DawoudSheraz** [comment]
Hello, cross-posting the update from relevant tutor-indigo issue https://github.com/overhangio/tutor-indigo/issues/107#issuecomment-2517251756

cc: @magajh @mariajgrimaldi

**Comment by DawoudSheraz** [comment]
Hello, adding some updates on dark theme fixups:
- Teams tab issues have been fixed in https://github.com/overhangio/tutor-indigo/pull/114
- Notes and textbook tabs have been fixed in https://github.com/overhangio/tutor-indigo/pull/115
- Course updates and handouts have been fixed in https://github.com/overhangio/tutor-indigo/pull/112/
- drop down problem has been fixed in https://github.com/overhangio/tutor-indigo/pull/110
- wiki issue was not reproducible

Pending issues (for a follow-up ticket)
- Discussions MFE side bars
- image-explorer 

I will add the relevant new issues once created. This would be good to close as we have covered major parts of the issue.

**Comment by DawoudSheraz** [comment]
I forgot to add that Edly has created a master issue for various indigo updates which also includes the dark theme updates https://github.com/overhangio/tutor-indigo/issues/116. The image explorer is being planned to be fixed at https://github.com/overhangio/tutor-indigo/issues/120. Discussion MFE might not be immediately fixed as it is pending on the addition of plugin slots in it. We might think of adding it ourselves if there are not plans to add it in near term.

**Comment by mariajgrimaldi** [comment]
Do you have any updates on this issue, @DawoudSheraz?

**Comment by mariajgrimaldi** [comment]
@magajh @efortish @BryanttV: can you help me confirm that these issues were fixed so we can close this? Thanks!

**Comment by efortish** [comment]
Hello @mariajgrimaldi I reviewed all features but discussions, everything seems good!
